lib/index.js

```diff
@@ -40,10 +39,11 @@ function build(worksheets) {
   var workBook = new _workbook2.default();
   worksheets.forEach(function (worksheet) {
     var name = worksheet.name || 'Sheet';
-    var data = (0, _helpers.buildSheetFromMatrix)(worksheet.data || [], worksheet.options || {});
+    var data = (0, _helpers.buildSheetFromMatrix)(worksheet.data || [], worksheet.options ? worksheet.options : worksheets.length === 1 ? options || {} : {}); // If worksheet has options, use this one ; if there's only one worksheet, use global options (for retro-compatibility)
```

Comment:
Looks good!! But do you think the worksheet.length comparison is necessary?
It would be enough that there is no worksheet.option.

Reply:
The idea was "better no options than a merging occurring on all worksheets" (unless there is one worksheet).

As you wish guys :-)

Reply:
The idea was "better no options than a merging occurring on all worksheets" (unless there is one worksheet).

As you wish guys :-)